io/src/main/scala/sbt/internal/io/Milli.scala

```diff
@@ -224,20 +224,6 @@ private object Linux32Milli extends PosixMilliIntUtim[Linux32] {
   }
 }
 
-private class FreeBSD64FileStat extends StatLong(120, 40, 48)
```

Comment:
Mima seems to have problems with private classes and traits inside of an object. If you add the missing classes to the mimaBinaryIssueFilters setting, CI should pass.

Reply:
Thanks, fixed.